.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,69 @@
+name: '🐞 버그 리포트'
+description: '버그를 리포트하기 위한 템플릿입니다'
+labels: ['bug']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        버그를 보고해 주셔서 감사합니다! 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)